### PR TITLE
2.0.1 - Minor bug fixes.

### PR DESCRIPTION
**Pull request by JakeMakesStuff**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "magiccap",
-    "version": "2.0.0",
+    "version": "2.0.1",
     "description": "A cross platform image capturing suite.",
     "main": "./dist/initialiser.js",
     "author": "Jake Gealer & Rhys O'Kane",
```
Comment (MattIPv4) [inline package.json]: Ngl, I feel I've been missed here?

Comment (JakeMakesStuff) [reply package.json]: Yeah that's legacy, I'm just going to make that "MagicCap Development Team" to account for the future.

**Comment by MattIPv4** [comment]
Why do we have a branch protection rule of two approving reviews?

**Comment by JakeMakesStuff** [comment]
@MattIPv4 *had - yeah, it was pretty dumb lol

**Pull request merged**